dashboard/app/assets/stylesheets/trophy_sparkle.scss

```diff
@@ -0,0 +1,39 @@
+.sparkle {
```

Comment:
i'm assuming there is a reason these styles are separate from the similar styles you added in `shared/`, but is there a reasonable way to de-duplicate since they're so similar?

Reply:
We can do shared SCSS in the repo's `/shared` directory, but given how small it was, how presumably temporary this styling is, and how we had to tweak the offsets for each sparkle, I decided to keep it separate for now. :)